bundle-workflow/scripts/default/integtest.sh

```diff
@@ -86,4 +90,8 @@ then
   PASSWORD=`echo $CREDENTIAL | awk -F ':' '{print $2}'`
 fi
 
+if [ -n "$DIRECTORY" ]
+then
+  cd $DIRECTORY
+fi
```

Comment:
we should not be changing directory inside integtest.sh. It can be handled outside of it. In the subprocess command to run shell script integtest.sh, you can specify what is the working directory to run the command from.